src/test/scriptnum_tests.cpp

```diff
@@ -180,18 +196,28 @@ BOOST_AUTO_TEST_CASE(operators)
     {
         for(size_t j = 0; j < sizeof(offsets) / sizeof(offsets[0]); ++j)
         {
+            assert(offsets[j] != INT64_MIN);
+
             RunOperators(values[i], values[i]);
-            RunOperators(values[i], -values[i]);
-            RunOperators(values[i], values[j]);
-            RunOperators(values[i], -values[j]);
-            RunOperators(values[i] + values[j], values[j]);
-            RunOperators(values[i] + values[j], -values[j]);
-            RunOperators(values[i] - values[j], values[j]);
-            RunOperators(values[i] - values[j], -values[j]);
-            RunOperators(values[i] + values[j], values[i] + values[j]);
-            RunOperators(values[i] + values[j], values[i] - values[j]);
-            RunOperators(values[i] - values[j], values[i] + values[j]);
-            RunOperators(values[i] - values[j], values[i] - values[j]);
```

Comment:
Note that the old code was broken, indexing `values[j]` instead of `offsets[j]`.

Reply:
In #4740 I assumed the code in the body was as intended, and changed the loop to be over `values`. Either interpretation is correct, but I think `values` provides slightly better coverage of corner cases (I also checked that the test would pass with `offsets`).